README.md

```diff
@@ -1,11 +1,11 @@
-# Stub Package for Launching a New EMP
+# Stub Package for Launching a New Perpetual
 
-The purpose of this repository/package is to make it easy to customize your EMP deployment. Feel free to use this
+The purpose of this repository/package is to make it easy to customize your Perpetual deployment. Feel free to use this
 repository in place or fork and customize it.
 
 ## Install system dependencies
 
-You will need to install nodejs v12 (we recommend `nvm` to manage node versions) and yarn.
+You will need to install nodejs (we recommend the latest stable version, nodejs v14, and `nvm` to manage node versions) and yarn.
```

Comment:
Some of the changes, like this one, could be made in the launch-emp repo too -- could you open a PR for them?

Reply:
I did! That one was waiting on the Kovan release, so we could test, but I'll take it out of draft mode, soon.